pycaret/loggers/dagshub_logger.py

```diff
@@ -0,0 +1,94 @@
+import os
+from pathlib import Path
+
+from pycaret.loggers.mlflow_logger import MlflowLogger
+
+try:
+    import dagshub
+    import mlflow
+    from dagshub.upload import Repo
+except ImportError:
+    dagshub = None
+
+
+class DagshubLogger(MlflowLogger):
+    def __init__(self, remote=None) -> None:
```

Comment:
In general, to avoid serialization issues, we should keep any actual initialisation logic in `init_experiment`. Ideally, `__init__` would just set attributes.

Reply:
Will fix!